contrib/exportLUT.lua

```diff
@@ -0,0 +1,126 @@
+--[[
+    This file is part of darktable,
+    copyright (c) 2020 Noah Clarke
+
+    darktable is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    darktable is distributed in the hope that it will be useful,
+     but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with darktable.  If not, see <http://www.gnu.org/licenses/>.
+]]
+--[[
+Add the following line to .config/darktable/luarc to enable this lightable module: 
+  require "exportLut"
+
+Given a haldCLUT identity file this script generates haldCLUTS from all the user's
+styles and exports them to a location of their choosing.
```

Comment:
Could you provide a link to an explanation of a haldCLUT identity file?  I didn't know what one was but I found https://rawpedia.rawtherapee.com/Film_Simulation, which provided an explanation and a way to create one.

Reply:
Here's a direct link to the identity haldCLUT: https://rawpedia.rawtherapee.com/images/8/83/Hald_CLUT_Identity.tif

Ideally I feel like it would be better from a usability standpoint to include it with the script. I wasn't able to find a clearly defined way to include extra resources with a script, however, so I left it up to the user to provide their own identity haldCLUT.